jax/_src/lax/lax.py

```diff
@@ -551,7 +551,6 @@ def conv_general_dilated(
   If `dimension_numbers` is `None`, the default is `('NCHW', 'OIHW', 'NCHW')`
   (for a 2D convolution).
   """
-  dnums: ConvDimensionNumbers
```

Comment:
did you mean to delete this?

Reply:
Yes, AFAIK this should not be needed since I have annotated the return type of `conv_dimension_numbers` function